Assets/Project/Scripts/Common/Managers/AddressableAssetManager.cs

```diff
@@ -17,6 +19,8 @@ public static class AddressableAssetManager
         /// </summary>
         private static bool _initialized = false;
 
+        public static readonly Subject<AsyncOperationHandle> OnAssetStartLoad = new Subject<AsyncOperationHandle>();
+
```

Comment:
カプセル化するために、`onNext` ができてしまう `Subject` は外部公開しないようにしましょう。
```suggestion
        private static readonly Subject<AsyncOperationHandle> OnAssetStartLoadSubject = new Subject<AsyncOperationHandle>();
        
        public static readonly IObservable<AsyncOperationHandle> OnAssetStartLoad = OnAssetStartLoadSubject.AsObservable();
```

Reply:
なるほど・・